app/views/layouts/_header.html.erb

```diff
@@ -7,15 +7,12 @@
   <%= stylesheet_link_tag "I18n" %>
 <% end %>
 
-<nav id="header" class="navbar navbar-inverse navbar-fixed-top">
-  <div class="container">
+<nav id="header" class="navbar navbar-expand-sm navbar-inverse bg-inverse">
     <!-- ||| menu, displayed for mobile: -->
-    <div class="navbar-header">
-      <a class="navbar-brand" id="brand" href="<%= root_path %>">Public Lab</a>
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#header-navbar-collapse">
-        <span class="fa fa-white fa-bars"></span>
-      </button>
-    </div>
+  <a class="navbar-brand" id="brand" href="<%= root_path %>">Public Lab</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#header-navbar-collapse">
```

Comment:
This is the button which should have disappeared in large screens, but it didn't.